lopper/assists/baremetal_gentestapp_xlnx.py

```diff
@@ -248,6 +286,8 @@ def xlnx_generate_testapp(tgt_node, sdt, options):
         for app in testapp:
             if 'SelfTest' in app or 'selftest' in app:
                 status_assignment = f"status = {app}({xpar_def});"
+            elif node == "axi_ethernet":
+                status_assignment = f"status = {app}(&{node}, &{dma_label}, {xpar_def});"
```

Comment:
what if dma_label is not present should we assign some dummy entry or through proper error?